Runtime/Managers/DeleteObjectsManager.cs

```diff
@@ -0,0 +1,57 @@
+using UnityEngine;
+using ReupVirtualTwin.managerInterfaces;
+using ReupVirtualTwin.helpers;
+using ReupVirtualTwin.helperInterfaces;
+using ReupVirtualTwin.enums;
+using ReupVirtualTwin.dataModels;
+using System.Collections;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using ReupVirtualTwin.controllerInterfaces;
+
+namespace ReupVirtualTwin.managers
+{
+    public class DeleteObjectsManager : MonoBehaviour, IDeleteObjectsManager
+
+    {
+        private IMediator _mediator;
+        public IMediator mediator { set { _mediator = value; } }
+        private ITagsController _tagsController;
+        public ITagsController tagsController { set => _tagsController = value; }
+
+        public bool AreWrappedObjectsDeletable(ObjectWrapperDTO wrapperDTO)
+        {
+            if (wrapperDTO == null || wrapperDTO.wrapper == null)
+            {
+                throw new ArgumentException("Selection wrapper is null");
+            }
+            else if (wrapperDTO.wrappedObjects.Count == 0)
+            {
+                throw new ArgumentException("There are no objects selected");
+            }
+            else if (!CheckTag(wrapperDTO.wrappedObjects))
+            {
+                throw new ArgumentException("Not all selected objects are deletable");
+            }
+            else
+            {
+                return true;
+            }
+        }
+
+        public void DeleteSelectedObjects(ObjectWrapperDTO wrapperDTO)
```

Comment:
I think we don`t need a whole wrapper but just a list of objects, we are even calling this method when the wrapper was destroyed
```suggestion
        public void DeleteSelectedObjects(List<GameObject> objectsToDelete)
```